Cargo.toml

```diff
@@ -39,6 +39,7 @@ global-context-less-secure = []
 secp256k1-sys = { version = "0.4.2", default-features = false, path = "./secp256k1-sys" }
 bitcoin_hashes = { version = "0.10", optional = true }
 rand = { version = "0.6", default-features = false, optional = true }
+# Our custom serde code requires an allocator, enable secp256k1 feature `std` or `alloc`.
```

Comment:
Where does it allocate? I don't see any such instance.

Besides, TODO after MSRV bump: use [crate renaming trick](https://github.com/LNP-BP/ln-types/blob/f6e7da9155f57e32d66f9e43eb13fd1d261b4c87/Cargo.toml#L18) instead.

Reply:
Allocates due to use of the `Vec` in `visit_seq` method.

Reply:
Ah, I see. Looks like using array is feasible?

Reply:
> Besides, TODO after MSRV bump: use [crate renaming trick](https://github.com/LNP-BP/ln-types/blob/f6e7da9155f57e32d66f9e43eb13fd1d261b4c87/Cargo.toml#L18) instead.

Not sure what you mean here mate, is this comment still relevant with the removal of alloc/vec stuff?

Reply:
It's not.